### ISSUE TITLE
Standardize Shadows

### ISSUE THREAD

**Issue by derek-boman** [mxenabled/mx-react-components#279]
We currently have one box shadow property however we are not being consistent in using it and there is a need for additional shadow constants.

My proposal is here: https://app.frontify.com/d/KHQZnlj1XMIO/web-patterns#/general/shadows-layering

We should use these on new projects. I'll create tasks for cleanup to existing products as I find them.

**Comment by jmophoto** [comment]
To pull the specs from the frontify link. 

```
.shadow-low {
  box-shadow: 0 1px 2px rgba(0,0,0,0.1); 
}

.shadow-med {
  box-shadow: 0 2px 6px rgba(0,0,0,0.1); 
}
.shadow-high {
  box-shadow: 0 10px 30px 5px rgba(0,0,0,0.1); 
}
```

**Comment by derek-boman** [comment]
btw feel free to format class names according to your standards, above is merely to share values.

**Comment by jmophoto** [comment]
These constants are now included in version 3.1.5.

- ShadowLow
- ShadowMed
- ShadowHigh
